docs/spire-workload-registrar.md

```diff
@@ -25,41 +83,87 @@ This requires few steps:
   spire-registrar-77ff576ccb-tsr4t   1/1     Running   0          0h24m   172.30.40.199   10.38.240.214   <none>           <none>
   ```
   In this case the registrar is running on the node `10.38.240.214`
+
+  There are 2 methods to register Workload Registrar:
+
+### Register Workload Registrar using Tornjak UI
 * Connect to Tornjak server UI, and list the agents.
 Get the SVID (SPIFFE ID) of the agent running on the specific node (as above). You can use the search function.
-* In the selected agent row in the table, click on the three dots in the rightmost column under `Workload Attestor Plugin`.  Under `Add WorkLoad Attestor Info→WorkLoad Attestor Plugin`, select Kubernetes, and click `Save & Add`.  
+* In the selected agent row in the table, click on the three dots in the rightmost column under `Workload Attestor Plugin`.  Under `Add WorkLoad Attestor Info→WorkLoad Attestor Plugin`, select Kubernetes, and click `Save & Add`.
 * Create a new entry `(Select Entries → Create Entry)`
    - Select the matching Agent SVID as a `Parent ID`
    - Use the sample string suggested at the end of the helm/OpenShift deployment as `SPIFFE ID` for the registrar:
    ```
-   spiffe://openshift.space-x.com/space-x.01/workload-registrar
+   spiffe://openshift.space-x.com/mycluster/workload-registrar
    ```
    - Under Selectors Recommendation, select the `selectors` suggested by the installation under `Selectors Recommendation`.  For example, if the installation suggests the following:
    ```
    k8s:sa:spire-k8s-registrar,k8s:ns:spire,k8s:container-name:k8s-workload-registrar
    ```
-   check off `k8s:sa`, `k8s:ns`, `k8s:container-name`.  Then under `Selectors`, fill in the suggested values.  
+   check off `k8s:sa`, `k8s:ns`, `k8s:container-name`.  Then under `Selectors`, fill in the suggested values.
    - Make sure to check the `Admin Flag`, so the registrar gets enough permissions to create new entries.
+* If you anticipate Workload Registrar pod might get recreated on a different node,
+create entries (`workload-registrar1`,`workload-registrar1`...) for every Agent Parent ID.
 
 If everything was fine, we should start seeing new entries, including the agents and the registrar `(Entries → Entry List)`
 Otherwise, review the registrar logs.
 
+### Register Workload Registrar manually
+Get to the cluster that is hosting the Tornjak/SPIRE Server,
+find the `spire-server-0` pod and get inside:
+
+```console
+kubectl -n tornjak get pods
+kubectl -n tornjak exec -it spire-server-0 exec -- sh
+```
+
+List all the agent objects, and find the spiffeID of the agent
+that is on the same node as the Registrar. This will be our `parentID`:
+
+```console
+/opt/spire/bin/spire-server agent list -registrationUDSPath /run/spire/sockets/registration.sock
```

Comment:
`-registrationUDSPath` is deprecated; some example that uses `-socketPath` would be good in case the command fails.